### PR TITLE
Fixed asm.bytes on by default in cursor mode

### LINKED ISSUE #9847: When toggling cursor, show bytes
When cursor is enabled enable asm.bytes if its not set. Maybe this should be the default behaviour, or maybe not, but will be good to be able to do that and add a config var to choose

### PR DESCRIPTION
**Pull request by cyanpencil**
Closes #9847 

__EDIT__: 

Remade this pr, now press # to toggle bytes when in disasm view. 
No more turning on by default when switching to cursor.

-----

__OLD_PR__:

When toggling the cursor, `asm.bytes` goes on and off.

However, if the user had specified before that `e asm.bytes = true`, then bytes are kept being shown even when exiting cursor mode.

This is done in a little hackish way:
 `asm.bytes = asm.bytes + 1` when switching to cursor mode
 `asm.bytes = asm.bytes - 1` when switching off cursor mode
(this means that `asm.bytes = 2` when it was set by the user and then switched to cursor mode),
 but doing like this avoids adding another config variable.

Tell me if it's too confusing.

### CODE REVIEW

**Comment by radare** [comment]
asm.bytes is boolean type, what about toggling it instead?

**Comment by radare** [comment]
anyway, maybe we should discuss for a better way to do that. because we want to get the bytes or not? because maybe the cursor in disasm without bytes should just scroll around instructions and ignore bytes :?

**Comment by cyanpencil** [comment]
> asm.bytes is boolean type, what about toggling it instead?

I decided not to toggle it because if the user wanted to see bytes in disasm without the cursor, then entering and exiting cursor mode will disable the bytes (and that would be something the user will not expect nor want). This way it "remembers" the previous state and won't disable bytes when exiting cursor mode if it was enabled before by the user.

However I understand that the implementation is a bit confusing, maybe I can add a config variable to be more clear in the code. 

> anyway, maybe we should discuss for a better way to do that. because we want to get the bytes or not? because maybe the cursor in disasm without bytes should just scroll around instructions and ignore bytes :?

Reading issue #9847 I assumed that bytes should be always on when entering cursor mode, but if I misunderstood tell me so I'll close this pr and continue discussion on the issue page.

**Comment by cyanpencil** [comment]
Imho that's not a bad idea, we could use for example `I` key to toggle bytes (both in visual and in graph), and leave them on by default like they are right now.

**Comment by radare** [comment]
I is already used. but not in the help message of V?

**Comment by cyanpencil** [comment]
`I` is not listed in the V? help maybe because it is an alias of `i`...?

Anyway. do you have any suggestions? Maybe a special character like `~` or `'`, or even `#`?

**Comment by radare** [comment]
nope. i = write, I = insert

**Comment by radare** [comment]
can you update the help message to add ‘I’ ?

about which char to use for toggling bytes.. im not sure :?

not sure if wW is anything useful.. we should have a discussion on all the viusal keys imho

File: libr/core/visual.c [hunk 1]
```diff
@@ -2484,6 +2486,9 @@ R_API int r_core_visual_cmd(RCore *core, const char *arg) {
 		case ')':
 			rotateAsmemu (core);
 			break;
+		case '#':
+			r_core_cmd0 (core, "e!asm.bytes");
```
Comment (radare) [inline libr/core/visual.c]: use the api: r_config_toggle(core->config, "asm.bytes");

**Comment by radare** [comment]
sorry for my late review. change this and i'll merge. thanks!!

**Pull request merged**